src/components/Calculator.js

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import Item from './Item';
+
+function Calculator() {
```

Comment:
- [ ] Kindly convert your component to a class-based component, this is a project requirement.